pgbadger

```diff
@@ -1320,7 +1320,7 @@ if ( $outdir && $retention && ($saved_last_line{datetime} || $pgb_saved_last_lin
 				}
 			}
 		}
-		# Now removed the corresponding days 
+		# Now removed the corresponding days
```

Comment:
Oh, do you really want to clear all trailing whitespaces ? This will make you the last author of all of these lines ;-)

Reply:
Oeps. My vim auto did that.

Reply:
Will fix

Reply:
Alright, should be fixed now.